ibu-imager/seedcreator/seedcreator.go

```diff
@@ -500,19 +495,42 @@ func (s *SeedCreator) waitTillOvnKubeNodeIsDown(ctx context.Context) {
 	defer deadlineCancel()
 }
 
-func (s *SeedCreator) getCatalogImages(ctx context.Context) ([]string, error) {
+// filterCatalogImages filters catalog source images as catalog sources have pull always policy
+// and there is no point to precache them.
+// List of catalog images are taken from catalog sources that are currently configured on cluster and known ocp ones.
+// Filtering known images will allow us to fix the case where those images were pulled in seed
+// and after it their catalog sources were removed but images are still part of podman images list as we create seed
+func (s *SeedCreator) filterCatalogImages(ctx context.Context, images []string) ([]string, error) {
+	defaultCatalogsImages := []string{"registry.redhat.io/redhat/community-operator-index",
+		"registry.redhat.io/redhat/redhat-operator-index",
+		"registry.redhat.io/redhat/certified-operator-index",
+		"registry.redhat.io/redhat/redhat-marketplace-index",
+	}
+
 	s.log.Info("Searching for catalog sources")
-	var images []string
+	var catalogImages []string
+
 	catalogSources := &operatorsv1alpha1.CatalogSourceList{}
 	allNamespaces := runtime.ListOptions{Namespace: metav1.NamespaceAll}
 	if err := s.client.List(ctx, catalogSources, &allNamespaces); err != nil {
 		return nil, fmt.Errorf("failed to list all catalogueSources %w", err)
 	}
 
 	for _, catalogSource := range catalogSources.Items {
-		images = append(images, catalogSource.Spec.Image)
+		// Splitting on : is technically wrong here because of port numbers
+		// but in this circumstance the code is still correct despite that
+		if !lo.Contains(defaultCatalogsImages, strings.Split(catalogSource.Spec.Image, ":")[0]) {
+			catalogImages = append(catalogImages, catalogSource.Spec.Image)
+		}
 	}
 
+	s.log.Infof("Removing list of catalog images from full image list, catalog images to remove %s",
```

Comment:
TBH, I'm not sure we want to log the images we might delete.
If we delete an image let's log that.